### ISSUE TITLE
Nx-ovloader 

### ISSUE THREAD

**Issue by 0zram** [WerWolv/nx-ovlloader#34]
Hi,

Nx-ovloader make atmosphere crash with both updated.
Is this problem normal and do you know any solution ?

**Comment by masagrator** [comment]
It's not normal since it works for everybody else.

**Comment by 0zram** [comment]
Do you have a solution ?

**Comment by ericalas93** [comment]
did you find a solution for this? i ended up deleting the folder since it kept crashing..

**Comment by 0zram** [comment]
No I didn't, i nearly brick my switch because I am novice, so I didn't try again. But some people told me that it exist an alternative to nx-ovloader so maybe try that (i didn't remember the name of it)

**Comment by ericalas93** [comment]
> No I didn't, i nearly brick my switch because I am novice, so I didn't try again. But some people told me that it exist an alternative to nx-ovloader so maybe try that (i didn't remember the name of it)

are you using a mac?

**Comment by 0zram** [comment]
Yeah, I know it's one of the causes of my probleme but I don't really have a choice

**Comment by ericalas93** [comment]
> Yeah, I know it's one of the causes of my probleme but I don't really have a choice

don't blame ya, I also use mac (and no access to a PC)

the issue was fixed by this: https://rentry.org/archivebitfix

ensure you have the two pieces on your sd, and then run the archive bit fix

Steps for me:
1. https://rentry.org/teslaoverlay#getting-started-setup download in move to your SD card both of "Tesla-Menu" and "nx-ovlloader" - ensure you are just copying the contents and not replacing full folders 
2. eject SD card
3. open hekate and run https://rentry.org/archivebitfix
4. 🎉
